tensorboard/plugins/mesh_visualizer/tf_mesh_dashboard/tf-mesh-dashboard.html

```diff
@@ -0,0 +1,279 @@
+  A frontend that displays a set of tf-mesh-loaders.
+-->
+<dom-module id="mesh-dashboard">
+  <template>
+    <tf-dashboard-layout>
+      <div class="sidebar all-controls">
+        <div class="sidebar-section">
+          <tf-runs-selector selected-runs="{{_selectedRuns}}">
+          </tf-runs-selector>
+        </div>
+        <div class="sidebar-section view-control">
+            <h3 class="title">Point of view</h3>
+            <div>
+              <paper-radio-group id="view-radio-group" selected="{{_selectedView}}">
+                <paper-radio-button id="all-radio-button" name="all">
+                  Display all points
+                </paper-radio-button>
+                <paper-tooltip animation-delay="0" for="all-radio-button" position="right" offset="0">
+                  Zoom and center camera to display all points at once. Note, that some points could
+                  be too far (i.e. too small) to be visible.
+                </paper-tooltip>
+                <paper-radio-button id="user-radio-button" name="user">
+                  Current view
+                </paper-radio-button>
+                <paper-tooltip animation-delay="0" for="user-radio-button" position="right" offset="0">
+                  Keep current camera position and zoom level.
+                </paper-tooltip>
+                <paper-radio-button id="user-radio-button" name="share">
+                  Share viewpoint
+                </paper-radio-button>
+                <paper-tooltip animation-delay="0" for="user-radio-button" position="right" offset="0">
+                  Share viewpoint among all cameras.
+                </paper-tooltip>
+              </paper-radio-group>
+            </div>
+        </div>
+      </div>
+      <div class="center">
+        <template is="dom-if" if="[[_dataNotFound]]">
+          <div class="no-data-warning">
+            <h3>No point cloud data was found.</h3>
+            <p>Probable causes:</p>
+            <ul>
+              <li>You haven’t written any point cloud data to your event files.
+              <li>TensorBoard can’t find your event files.
+            </ul>
+            <p>
+            If you’re new to using TensorBoard, and want to find out how
+            to add data and set up your event files, check out the
+            <a href="https://github.com/tensorflow/tensorboard/blob/master/README.md">README</a>
+            and perhaps the <a href="https://www.tensorflow.org/get_started/summaries_and_tensorboard">TensorBoard tutorial</a>.
+            <p>
+            If you think TensorBoard is configured properly, please see
+            <a href="https://github.com/tensorflow/tensorboard/blob/master/README.md#my-tensorboard-isnt-showing-any-data-whats-wrong">the section of the README devoted to missing data problems</a>
+            and consider filing an issue on GitHub.
+          </div>
+        </template>
+        <template is="dom-if" if="[[!_dataNotFound]]">
+          <paper-input
```

Comment:
Instead of paper-input and iron-icon here, I would use https://github.com/tensorflow/tensorboard/blob/master/tensorboard/components/tf_categorization_utils/tf-tag-filterer.html

Example of usage: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html#L130